### ISSUE TITLE
Sound improvements - update wind sound

### ISSUE THREAD

**Issue by haggen88** [rafaelmardojai/blanket#147]
Hello. I would like to know if it is possible to improve the sound of the current wind, it seems to me that the current sound [whose loop seems a bit intense, somewhat saturated and with a poor range] can be improved to one that allows more and with little concentration and / or relaxation.  

Here I found some alternatives, hopefully @Porrumentzio can give his opinion regarding the LUFS analysis and playback adjustments / edits: 

- https://freesound.org/people/felix.blume/sounds/217506/  > Wind blowing in a field in Texas, USA   by  felix.blume   [CC0 1.0]
 >  [wind blowing in a field.zip](https://github.com/rafaelmardojai/blanket/files/6919345/wind.blowing.in.a.field.zip)

- https://freesound.org/people/Dynamicell/sounds/17553/  > Wind_Howling_NIghttime    by   Dynamicell   [CC BY 3.0]
 >  [wind howling.zip](https://github.com/rafaelmardojai/blanket/files/6919349/wind.howling.zip)

Best regards

**Comment by Porrumentzio** [comment]
**wind blowing in a field**: it's OK, volume must be decreased but alright.
**wind howling**: not OK, too small dynamic range
**johanwestling**: could be good. Low volume but it could fit, even if dynamic range is not very large (4,1LU)
**nickydunne**: not good, it's too monotone.
**Cinetony**: really good, needs increasing volume.

I would definitely go for first or last, as they have bird or wood creaking sounds and make the clip less monotone. Their dynamic range are good and volumes must be fitted.

**Comment by rafaelmardojai** [comment]
I kinda prefer the first if we're talking about a replacement. It's a more "pure" wind sound IMO.

**Comment by haggen88** [comment]
@Porrumentzio I agree "wind blowing in a field" is more suited to the wind profile, it also combines better with other sounds like rain and thunder. 
I leave it up to you.

PS: I guess you have to make small adjustments to the clip to fit a loop without dramatic cuts.

**Comment by rafaelmardojai** [comment]
Is a yes from my side, so @Porrumentzio if you have time to help on this would be awesome!

**Comment by haggen88** [comment]
BTW @rafaelmardojai @Porrumentzio  Is there any other sounds you want to update / replace?

**Comment by rafaelmardojai** [comment]
@haggen88  I think Summer night, Coffee Shop and Fireplace weren't ideal in the test @Porrumentzio did, but we didn't find a better replacement so we keep them.

**Comment by haggen88** [comment]
@rafaelmardojai  I will try to find audio files that may be useful  
[I will leave them in this thread].

**Comment by haggen88** [comment]
@Porrumentzio @rafaelmardojai  Here are some candidates for Fireplace:

- https://freesound.org/people/reinsamba/sounds/18766/ >  chimney fire by reinsamba [CC BY 3.0]
 >  [reinsamba - chimney fire.zip](https://github.com/rafaelmardojai/blanket/files/6995948/reinsamba.-.chimney.fire.zip)

- https://freesound.org/people/kyles/sounds/454016/ >  fire in fireplace close1 by kyles [CC0 1.0]
 >  [kyles - fire in fireplace close1.zip](https://github.com/rafaelmardojai/blanket/files/6995952/kyles.-.fire.in.fireplace.close1.zip)

- https://freesound.org/people/tosha73/sounds/513280/ >  Bonfire HQ by tosha73 [CC0 1.0]
 >  [tosha73 - Bonfire HQ.zip](https://github.com/rafaelmardojai/blanket/files/6995955/tosha73.-.Bonfire.HQ.zip)

- https://freesound.org/people/dobroide/sounds/120644/ >  20110503_fireplace.00 by dobroide [CC BY 3.0]
 >  [dobroide - 20110503_fireplace.00.zip](https://github.com/rafaelmardojai/blanket/files/6995971/dobroide.-.20110503_fireplace.00.zip)

- https://freesound.org/people/mah1taS/sounds/434026/ >  FireSound by mah1taS [CC0 1.0]
 >  [mah1taS - FireSound.zip](https://github.com/rafaelmardojai/blanket/files/6995975/mah1taS.-.FireSound.zip)

**Comment by haggen88** [comment]
@Porrumentzio @rafaelmardojai Here are some candidates for **Summer night**:

- https://freesound.org/people/dobroide/sounds/20575/ >  20060706.night.forest02 by dobroide [CC BY 3.0]
>  [dobroide - 20060706.night.forest02.zip](https://github.com/rafaelmardojai/blanket/files/7004240/dobroide.-.20060706.night.forest02.zip)

- https://freesound.org/people/eric5335/sounds/102129/ > Summer night by eric5335 [CC BY-NC 3.0]
>  [eric5335 - Summer night.zip](https://github.com/rafaelmardojai/blanket/files/7004241/eric5335.-.Summer.night.zip)

- https://freesound.org/people/hargissssound/sounds/319372/ > Cricket on Summer Night (binaural) by hargissssound [CC0 1.0]
>  [hargissssound - Cricket on Summer Night (binaural).zip](https://github.com/rafaelmardojai/blanket/files/7004242/hargissssound.-.Cricket.on.Summer.Night.binaural.zip)

- https://freesound.org/people/Moulaythami/sounds/536930/ >  Cricket Ambience, Remix, A by Moulaythami [CC BY 3.0]
>  [Moulaythami - Cricket Ambience, Remix, A.zip](https://github.com/rafaelmardojai/blanket/files/7004244/Moulaythami.-.Cricket.Ambience.Remix.A.zip)

- https://freesound.org/people/raoul_slayer/sounds/203598/ > CricketsandTawnyOwl by raoul_slayer [CC0 1.0]
>  [raoul_slayer - CricketsandTawnyOwl.zip](https://github.com/rafaelmardojai/blanket/files/7004245/raoul_slayer.-.CricketsandTawnyOwl.zip)

- https://freesound.org/people/tim.kahn/sounds/477865/ > On the Bank of the Little Missouri River 2 by tim.kahn [CC BY-NC 3.0]
>  [tim.kahn - On the Bank of the Little Missouri River 2.zip](https://github.com/rafaelmardojai/blanket/files/7004248/tim.kahn.-.On.the.Bank.of.the.Little.Missouri.River.2.zip)

**Comment by rafaelmardojai** [comment]
@Porrumentzio any comments? :P

**Comment by Porrumentzio** [comment]
I'm busy with other projects, I can't give time to this right now. You can choose some of them according to what you hear (which of them sounds better and more comfortable) and then, maybe I will have time to analyze them

**Comment by rafaelmardojai** [comment]
> I'm busy with other projects, I can't give time to this right now. You can choose some of them according to what you hear (which of them sounds better and more comfortable) and then, maybe I will have time to analyze them

Ok! No problem, I will check the sounds when I got some free time and let you know. I want to release a new Blanket version next week but there's no hurry on solving this for that.

**Comment by haggen88** [comment]
You have to avoid rushing, it would be good to wait for @Porrumentzio to calmly analyze the audios, otherwise I have 22 candidate samples to replace Coffee Shop [I have doubts that it really needs to be an adequate sound, I hope you guide me] ... I must also make a selection before publishing them.

Btw @rafaelmardojai something useful for the next launch would be to include a _circular blanket icon_ to maximize compatibility with icon themes ... for [example](https://github.com/rafaelmardojai/blanket/files/7110620/Blanket.circle.zip), this is how I have my desktop: 

![20 2](https://user-images.githubusercontent.com/68134588/132110005-d392a25d-911b-4cb4-a141-9c7f4dcfcee3.png)

**Comment by rafaelmardojai** [comment]
Yeah, I don't have intention to rush about this, this can be solved in a posterior release. But I have some nice new features on `master` that I don't want too keep holding.

> Btw @rafaelmardojai something useful for the next release would be to include a circular blanket icon to maximize compatibility with icon themes ... for example, this is how I have my desktop:

Hmm, I'm not sure, probably makes more sense asking the icons themes to support Blanket. Also for that to work I would need to copy that icon to specific icon theme locations what isn't great IMO.

**Comment by haggen88** [comment]
@Porrumentzio @rafaelmardojai Here are some candidates for Coffee Shop  [Btw.. perhaps in some clips it is necessary to use some effects (eg reverb & EQ) to cover up the speech of the people and that the words are not distinguished] :

- https://freesound.org/people/190042/sounds/487581/ > Coffee Shop Ambience by '190042' [CC BY 3.0]
[190042 - Coffee Shop Ambience.zip](https://github.com/rafaelmardojai/blanket/files/7548083/190042.-.Coffee.Shop.Ambience.zip)

- https://freesound.org/people/waweee/sounds/370973/ > coffee shop ambience by 'waweee' [CC0 1.0]
[waweee - coffee shop ambience.zip](https://github.com/rafaelmardojai/blanket/files/7548062/waweee.-.coffee.shop.ambience.zip)

- https://freesound.org/people/JanevdMerwe1995/sounds/327594/ > Ambience of Coffee-shop by 'JanevdMerwe1995' [CC BY-NC 3.0]
[JanevdMerwe1995 - Ambience of Coffee-shop.zip](https://github.com/rafaelmardojai/blanket/files/7548067/JanevdMerwe1995.-.Ambience.of.Coffee-shop.zip)

- https://freesound.org/people/elliotrambach/sounds/344473/ > Moderately Crowded Coffee Shop Ambiance by 'elliotrambach' [CC0 1.0]
[elliotrambach - Moderately Crowded Coffee Shop Ambiance.zip](https://github.com/rafaelmardojai/blanket/files/7548077/elliotrambach.-.Moderately.Crowded.Coffee.Shop.Ambiance.zip)

...

- https://freesound.org/people/JarredGibb/sounds/244915/ >  Small coffee shop quiet by 'JarredGibb' [CC0 1.0]
[JarredGibb -  Small coffee shop quiet.zip](https://github.com/rafaelmardojai/blanket/files/7548101/JarredGibb.-.Small.coffee.shop.quiet.zip)

- https://freesound.org/people/Nox_Sound/sounds/553888/ > Ambiance_Atmosphere_Coffee_People&Rain_Loop by 'Nox_Sound' [CC0 1.0]
[Nox_Sound - Ambiance_Atmosphere_Coffee_People&Rain_Loop.zip](https://github.com/rafaelmardojai/blanket/files/7548108/Nox_Sound.-.Ambiance_Atmosphere_Coffee_People.Rain_Loop.zip)

---
These are not from a cafe but because of the ambient audio I think they can work: 

- https://freesound.org/people/Juraj%20Balaz/sounds/404082/ > atmosphere in pub_01 by 'Juraj Balaz' [CC BY-NC 3.0]
[Juraj Balaz - atmosphere in pub_01.zip](https://github.com/rafaelmardojai/blanket/files/7548136/Juraj.Balaz.-.atmosphere.in.pub_01.zip)

- https://freesound.org/people/Erdie/sounds/165612/ >  pub-reverb by 'Erdie' [CC BY 3.0]
[Erdie - pub-reverb.zip](https://github.com/rafaelmardojai/blanket/files/7548138/Erdie.-.pub-reverb.zip)

**Comment by DancingGeek** [comment]
[fire.zip](https://github.com/user-attachments/files/28431775/fire.zip)
This is a clip of about 30 seconds taken from the sound suggested by @haggen88 that I thought was the cleanest and best to listen to - https://freesound.org/people/reinsamba/sounds/18766/ > chimney fire by reinsamba [CC BY 3.0]
I used Ardour to get it to meet the sound specs and it tests well to my ears.
